src/Agent.Service/Windows/Program.cs

```diff
@@ -12,33 +12,18 @@ static class Program
         /// </summary>
         static int Main(String[] args)
         {
-            if (args != null && args.Length == 1 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
+            if (args != null && args.Length == 1)
             {
-                // TODO: LOC all strings.
-                if (!EventLog.Exists("Application"))
+                if(args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
it is the old code. I shall check with Ting and then react. Will do it in further iterations.

---
In reply to: [115211563](https://github.com/Microsoft/vsts-agent/pull/951#discussion_r115211563) [](ancestors = 115211563)